messages/JsonMessages/CompanionTextures.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from "zod";
+import { extendApi } from "@anatine/zod-openapi";
+
+/*
+ * WARNING! The original file is in /messages/JsonMessages.
+ * All other files are automatically copied from this file on container startup / build
+ */
+
+//The list of all the companion textures
+
+export const companionTexture = z.object({
+    name: extendApi(z.string(), { description: "The name of the texture.", example: "dog1" }),
+    img: extendApi(z.string(), {
+        description: "The URL of the image of the texture.",
+        example: "http://example.com/resources/characters/pipoya/Cat 01-1.png",
+    }),
+    behaviour: extendApi(z.string(), { description: "The companion behavior (dog|cat)", example: "dog" }),
```

Comment:
An enum check was preferable here to a string check to avoid unknown string value. I have also add a simple behavior if we want implement another thing who isn't a dog or a cat.
```ts
behaviour: extendApi(z.enum(['simple', 'dog', 'cat']), { description: "The companion behavior (simple|dog|cat)", example: "dog" }),
```

Reply:
Done